idutils/__init__.py

```diff
@@ -16,1005 +16,26 @@
-    "MGP_AKRJ_",  # Mus musculus (Mouse AKR/J)
-    "ENSDAR",  # Danio rerio (Zebrafish)
-    "ENSMUS",  # Mus musculus (Mouse)
-    "ENSETE",  # Echinops telfairi (Lesser hedgehog tenrec)
-    "ENSSBO",  # Saimiri boliviensis boliviensis (Bolivian squirrel monkey)
-    "ENS",  # Homo sapiens (Human)
-    "ENSCGR",  # Cricetulus griseus (Chinese hamster CHOK1GS)
-    "ENSFCA",  # Felis catus (Cat)
-    "MGP_BALBcJ_",  # Mus musculus (Mouse BALB/cJ)
-    "MGP_PahariEiJ_",  # Mus pahari (Shrew mouse)
-    "ENSCSA",  # Chlorocebus sabaeus (Vervet-AGM)
-    "ENSCCA",  # Cebus capucinus imitator (Capuchin)
-    "ENSOAR",  # Ovis aries (Sheep)
-    "ENSCHI",  # Capra hircus (Goat)
-    "ENSDOR",  # Dipodomys ordii (Kangaroo rat)
-    "ENSCHO",  # Choloepus hoffmanni (Sloth)
-    "ENSSHA",  # Sarcophilus harrisii (Tasmanian devil)
-    "ENSMPU",  # Mustela putorius furo (Ferret)
-    "ENSNLE",  # Nomascus leucogenys (Gibbon)
-    "ENSXMA",  # Xiphophorus maculatus (Platyfish)
-    "ENSSSC",  # Sus scrofa (Pig)
-    "ENSEEU",  # Erinaceus europaeus (Hedgehog)
-    "ENSPSI",  # Pelodiscus sinensis (Chinese softshell turtle)
-    "MGP_DBA2J_",  # Mus musculus (Mouse DBA/2J)
-    "ENSAMX",  # Astyanax mexicanus (Cave fish)
-    "MGP_WSBEiJ_",  # Mus musculus domesticus (Mouse WSB/EiJ)
-    "ENSJJA",  # Jaculus jaculus (Lesser Egyptian jerboa)
-    "ENSCIN",  # Ciona intestinalis
-    "ENSPPA",  # Pan paniscus (Bonobo)
-    "MGP_SPRETEiJ_",  # Mus spretus (Algerian mouse)
-    "ENSCAN",  # Colobus angolensis palliatus (Angola colobus)
-    "MGP_NODShiLtJ_",  # Mus musculus (Mouse NOD/ShiLtJ)
-    "ENSCLA",  # Chinchilla lanigera (Long-tailed chinchilla)
-    "ENSCPO",  # Cavia porcellus (Guinea Pig)
-    "ENSDNO",  # Dasypus novemcinctus (Armadillo)
-    "ENSPFO",  # Poecilia formosa (Amazon molly)
-    "ENSMIC",  # Microcebus murinus (Mouse Lemur)
-    "MGP_FVBNJ_",  # Mus musculus (Mouse FVB/NJ)
-    "MGP_CBAJ_",  # Mus musculus (Mouse CBA/J)
-    "ENSSTO",  # Ictidomys tridecemlineatus (Squirrel)
-    "ENSRNO",  # Rattus norvegicus (Rat)
-    "ENSMOC",  # Microtus ochrogaster (Prairie vole)
-    "ENSTNI",  # Tetraodon nigroviridis (Tetraodon)
-    "ENSPPY",  # Pongo abelii (Orangutan)
-    "ENSGAC",  # Gasterosteus aculeatus (Stickleback)
-    "ENSLOC",  # Lepisosteus oculatus (Spotted gar)
-    "ENSODE",  # Octodon degus (Degu)
-    "ENSPCA",  # Procavia capensis (Hyrax)
-    "ENSECA",  # Equus caballus (Horse)
-    "ENSOAN",  # Ornithorhynchus anatinus (Platypus)
-    "MGP_CAROLIEiJ_",  # Mus caroli (Ryukyu mouse)
-    "ENSHGLM",  # Heterocephalus glaber (Naked mole-rat male)
-    "MGP_129S1SvImJ_",  # Mus musculus (Mouse 129S1/SvImJ)
-    "ENSRBI",  # Rhinopithecus bieti (Black snub-nosed monkey)
-    "ENSMLU",  # Myotis lucifugus (Microbat)
-    "ENSMLE",  # Mandrillus leucophaeus (Drill)
-    "ENSMFA",  # Macaca fascicularis (Crab-eating macaque)
-)
-"""List of species-specific prefixes for Ensembl accession numbers.
+def import_attributes():
```

Comment:
Please add a deprecation warning that we might remove this backwards compatible import mechanism in the future, and clearly explain that the imports should be changed.